jre_emul/tests.mk

```diff
@@ -335,6 +336,10 @@ $(TESTS_DIR)/core_plus_concurrent:
 	@mkdir -p $(@D)
 	$(J2OBJCC) -ljre_concurrent -ljre_util -o $@ -ObjC
 
+$(TESTS_DIR)/core_plus_file:
+	@mkdir -p $(@D)
+	$(J2OBJCC) -ljre_file -o $@ -ObjC
```

Comment:
The file subset doesn't have any dependencies on other subsets? If that's true, awesome!

Reply:
Not for now! I am still working on adding the missing sources (Unix related for example) and dependencies are probably going to appear.

Reply:
Got it, thanks.